src/engine/source/vdscanner/test/src/unit/versionMatcher_test.cpp

```diff
@@ -538,7 +538,7 @@ TEST_F(VersionMatcherTest, comparePEP440_AlternativeSyntax_OkGreater)
 
 TEST_F(VersionMatcherTest, comparePEP440_ErrorInvalidVersion)
 {
-    EXPECT_THROW(VersionMatcher::compare("A.B.C", "1!2.0b2.post345.dev456", VersionObjectType::PEP440),
+    EXPECT_THROW(VersionMatcher::compare("A.B~C", "1!2.0b2.post345.dev456", VersionObjectType::PEP440),
```

Comment:
I'd like to know if this change is right or not.
If it isn't, what should we need to improve?
If it is, what should we backport?

Reply:
Same comment before